icu4c/source/python/icutools/databuilder/filtration_schema.json

```diff
@@ -49,17 +49,25 @@
         "fileReplacements": {
             "type": "object",
             "properties": {
-                "directory": { "type": "string" },
+                "directory": {
+                    "type": "string",
+                    "pattern": "^(/|\\$SRC|\\$FILTERS|\\$CWD)?.*$"
```

Comment:
should the variable patterns each include a trailing `/` ?

Reply:
There is not expected to be a trailing slash.  Maybe it would be better to actually forbid one.